src/base/ceramicApiKit.ts

```diff
@@ -0,0 +1,197 @@
+import { SafeMultisigTransactionResponse } from "@safe-global/safe-core-sdk-types";
+import { ProposeTransactionProps } from "@safe-global/api-kit/dist/src/types/safeTransactionServiceTypes";
+import { definition } from "./ceramicModels";
+import { ethers } from "ethers";
+
+interface ConfirmationNode {
+  owner: string;
+  id: string;
+  signature: string;
+  signatureType: string;
+  submissionDate: string;
+  transactionHash: string;
+  confirmationType: string;
+}
+
+interface ConfirmationEdge {
+  node: ConfirmationNode;
+}
+
+interface ConfirmationIndexData {
+  data: {
+    confirmationIndex: {
+      edges: ConfirmationEdge[];
+    };
+  }
+}
+
+export class ceramicApiKit {
+  private composeClient: any;
+  private privateKey: string;
+  private ceramicUrl: string;
+
+  constructor(privateKey: string, ceramicUrl: string) {
+    this.privateKey = privateKey;
+    this.ceramicUrl = ceramicUrl;
+  }
+
+  async connect() {
+    try {
+      const { ComposeClient } = await import('@composedb/client');
+      const composeClient = new ComposeClient({
+        ceramic: this.ceramicUrl,
+        //cannot import type from ESM only module in nest.js
+        // @ts-ignore
+        definition: definition,
+      });
+      const { fromString } = await import('uint8arrays/from-string');
+      const seedArray = fromString(this.privateKey, "base16");
```

Comment:
Is it safe to use the same private key as the safewallet account?

Reply:
very difficult to answer, 
on the surface, these packages doesn't send private to anywhere, just sign signatures,
but I can get your concern，let's me do research and see if threre's  more secured solution

Reply:
let's use other key